e2e/testcases/workload_identity_test.go

```diff
@@ -261,6 +270,16 @@ func TestWorkloadIdentity(t *testing.T) {
 				tc.sourceChart = chart.Name
 				tc.sourceVersion = chart.Version
 				tc.rootCommitFn = nomostest.HelmChartVersionShaFn(chart.Version)
+			} else if tc.sourceType == v1beta1.OciSource && tc.requireOCIArtifactRegistry {
+				image, err := nt.BuildAndPushOCIImage(nomostest.RootSyncNN(rs.Name),
+					registryproviders.ImageInputDir("../testdata/hydration/kustomize-components"),
+					registryproviders.ImageVersion(tc.sourceVersion))
+				if err != nil {
+					nt.T.Fatalf("failed to push oci image: %v", err)
+				}
+
+				tc.sourceRepo = image.FloatingTag()
```

Comment:
Why is this variable value changing from the local directory path to an image tag? I think we should have two separate variables to keep this clear

Reply:
Added another variable for the sourcePackage